icu4j/APIChangeReport.html

```diff
@@ -127,6 +127,14 @@ <h3>Package com.ibm.icu.lang</h3>
 </ul>
 </ul>
 
+<h3>Package com.ibm.icu.message2</h3>
+<ul>
+Mf2DataModel.OrderedMap
+<ul>
+<li><span style='color:orange'>(draft)</span> public <i>Mf2DataModel.OrderedMap</i>()</li>
```

Comment:
@yumaoka I asked @mihnita to check, and this class is marked `@internal` and `@deprecated` like the rest of the new MessageFormatter tech preview. Why does this one alone show up as "draft" here?

Reply:
mihinta discovered the problem. Waiting on [pull#2194](https://github.com/unicode-org/icu/pull/2194) to fix a constructor, then will regenerated the ICU4C report.